src/experiments/rl_envs/JuliaRL_DQN_CartPole.jl

```diff
@@ -16,24 +16,21 @@ function RLCore.Experiment(
 
     env = CartPoleEnv(; T = Float32, rng = rng)
     ns, na = length(state(env)), length(action_space(env))
+    base_model = Chain(
+        Dense(ns, 128, relu; initW = glorot_uniform(rng)),
+        Dense(128, 128, relu; initW = glorot_uniform(rng)),
+        Dense(128, na; initW = glorot_uniform(rng))
+        )
 
     agent = Agent(
         policy = QBasedPolicy(
             learner = DQNLearner(
                 approximator = NeuralNetworkApproximator(
-                    model = Chain(
-                        Dense(ns, 128, relu; initW = glorot_uniform(rng)),
-                        Dense(128, 128, relu; initW = glorot_uniform(rng)),
-                        Dense(128, na; initW = glorot_uniform(rng)),
-                    ) |> cpu,
+                    model = build_dueling_network(base_model) |> cpu,
```

Comment:
Create the `DuelingNetwork` structure directly here.

Reply:
Besides, describe that the `DuelingNetwork` is used in the `description` string.